src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2059,6 +2059,10 @@ public void BuildAMassiveApp()
 					KnownPackages.GooglePlayServices_22_0_0_2,
 				},
 			};
+			//NOTE: under MSBuild BuildingInsideVisualStudio prevents the projects from being built as dependencies
+			if (sb.RunningMSBuild) {
+				app1.SetProperty("BuildingInsideVisualStudio", "False");
```

Comment:
Should we just *always* set this? I don't think that this will negatively impact `xbuild`.